packages/calcite-components/src/components/shell/shell.e2e.ts

```diff
@@ -122,3 +123,71 @@ describe("calcite-shell", () => {
     expect(panelTop).toBeNull();
   });
 });
+
+const tipManagerHTML = html`
+  <calcite-tip-manager>
+    <calcite-tip heading="Example tip title">
+      <calcite-link href="http://www.esri.com">An example link</calcite-link>
+    </calcite-tip>
+  </calcite-tip-manager>
+`;
+
+describe("theme", () => {
+  const shellHTML = html`
+    <calcite-shell>
+      ${tipManagerHTML}
```

Comment:
Nitpick: `tipManagerHTML` is only used once, so it could be inlined here.

Reply:
I'll inline it, although I feel like separating chunks makes it visually easier to parse through html.